spring-grpc-spring-boot-autoconfigure/src/test/java/org/springframework/grpc/autoconfigure/server/GrpcServletAutoConfigurationTests.java

```diff
@@ -90,7 +74,40 @@ void whenNoBindableServicesRegisteredAutoConfigurationIsSkipped() {
 
 	@Test
 	void whenWebApplicationServletIsAutoConfigured() {
-		this.contextRunner().run((context) -> assertThat(context).getBean(ServletRegistrationBean.class).isNotNull());
+		this.contextRunner().run((context) -> {
```

Comment:
I think we can leave this test the way that it was and verify default property values below.

Reply:
make sense. fixed in 2f50e5c27732722dd6027127135d1673f1503a59